templates/php-component/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "2"
+services:
+  # for development purposes
+  dev:
+    build: 
+      context: .
+      dockerfile: Dockerfile-tests
```

Comment:
Proč se to jmenuje `-tests`, když je to dev? A k čemu je to dobré na vývoj? Není to spíš pozůstatek z toho, když se build spouštěl přes `docker-compose`? Protože na to, aby se tím dalo debugovat to je ještě málo nastavené.

Reply:
je to malo nastavene a proc se to jmenuje -tests nevim :)

Reply:
asi to muzem nechat na step 2, at  to ted nekomplikujem

Reply:
OK, odeberu a hodím do sólo branche